src/singletons/Settings.hpp

```diff
@@ -292,6 +292,17 @@ class Settings : public ABSettings, public ConcurrentSettings
     QStringSetting firstMessageHighlightColor = {
         "/highlighting/firstMessageHighlightColor", ""};
 
+    BoolSetting enableElevatedMessageHighlight = {
+        "/highlighting/elevatedMessageHighlight/highlighted", true};
+    //    BoolSetting enableElevatedMessageHighlightSound = {
+    //        "/highlighting/elevatedMessageHighlight/enableSound", false};
+    //    BoolSetting enableElevatedMessageHighlightTaskbar = {
+    //        "/highlighting/elevatedMessageHighlight/enableTaskbarFlashing", false};
+    QStringSetting elevatedMessageHighlightSoundUrl = {
+        "/highlighting/elevatedMessageHighlightSoundUrl", ""};
+    QStringSetting elevatedMessageHighlightColor = {
+        "/highlighting/elevatedMessageHighlightColor", ""};
```

Comment:
Could you change the paths of these two to be under the `/highlighting/elevatedMessageHighlight/` "scope"? similar to the other 3 settings above (I know this is inconsistent with firstMessageHighlight)